spec/features/admin/admin_can_see_items_marked_retired.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature 'Logged in admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.